Resources/Textures/Clothing/Head/Hats/safarihat.rsi/meta.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "discord: checkraze#7501 github: cheakraze",
```

Comment:
please properly credit me. "Created by EmoGarbage404" is fine with CC-0.

Reply:
My apologies, I will change them when I get home, I forgot thats in the json

Reply:
changed thank you again EmoGarbage